jsk_baxter_robot/jsk_baxter_startup/jsk_baxter_lifelog/estimated_endeffector.l

```diff
@@ -5,27 +5,31 @@
 
 (if (not (boundp '*baxter*))
     (setq *baxter* (instance baxter-robot :init)))
+(setq frames (make-hash-table))
+(dolist (arm (list :rarm :larm))
+  (sethash arm frames (send (send (send *baxter* arm :end-coords) :parent) :name))
+  )
 
-(ros::rate 20)
+(ros::rate 100)
```

Comment:
I think this is a good PR. As a comment, if you include the reason for changing the loop rate in the code, it will help readers understand why this value was chosen.

Reply:
the 100 Hz is the same rate of Baxter's joint state. This is why I set the rate into 100 Hz.  
But I didn't mean to PR with this commit... It is just my mistake.
Is it better to separate the camera setting PR and end_effector PR?

Reply:
I see.
In that case, it's better to split the PR. Adding an explanation in the EusLisp code about the rate changes in the separated PR would be good.

Reply:
Okay, thanks you.
I will PR this changes again.